src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
```

Comment:
When a specific object is selected using `selectedObjectIndex` prop, this method converts it to picking color to be set as a uniform.